packages/patternfly-3/patternfly-react/CONTRIBUTING.md

```diff
@@ -15,95 +15,6 @@
-  ```text
-  packages/patternfly-3/patternfly-react/src/components/[ComponentName]/
-    index.js - Barrel File exporting public exports
-    ComponentName.js - Component Implementation
-    ComponentName.test.js - Component Tests
-    ComponentName.stories.js - Component Stories
-  ```
-
-4.  Develop your component. After development is complete, ensure tests and lint standards pass.
-
-```text
-$ yarn test
-```
-
-Ensure no lint errors are introduced in `yarn-error.log` after running this command.
-
-5.  Add a commit using `yarn commit`:
-
-This project uses [`lerna`](https://lernajs.io/) to do automatic releases and generate a changelog based on the commit history. So we follow [a convention][3] for commit messages. Please follow this convention for your commit messages.
-
-You can use `commitizen` to help you to follow [the convention][3].
-
-Once you are ready to commit the changes, please use the below commands:
-
-```text
-$ git add <files to be committed>
-$ yarn commit
-```
-
-... and follow the instruction of the interactive prompt.
-
-6.  Rebase
-
-Use `git rebase` (not `git merge`) to sync your work from time to time. Ensure all commits related to a single issue have been [squashed](https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit).
-
-```text
-$ git fetch upstream
-$ git rebase upstream/master
-```
-
-7.  Push
-
-```text
-$ git push origin my-branch
-```
-
-8.  Create a Pull Request
-
-[Open a pull request](https://help.github.com/articles/using-pull-requests/) with a clear title and description against the `master` branch. Please be sure to include all of the following in your PR:
-
-- Any relevant issues associated with this pull request (`enhancement` issues, `bug` issues, etc.)
-- Storybook and Documentation
-  - Include a link to the design documentation in the [PatternFly Pattern Library](http://www.patternfly.org/pattern-library/) if it exists. If a PatternFly design does not exist yet, then provide a description that explains when the component would be used and what goal or task it helps to accomplish.
-
-A link to the PatternFly 3 Storybook will be automatically generated and posted as a comment after the pull request build is complete.
-
-Once your pull request has been reviewed, if all conditions above have been met your pull request will be approved and merged.
-
-Please help in ensuring all relevant issues are closed and that any subsequent issues needed have been noted with this pull request.
-
 ## Additional Information
 
-See the PatternFly React Guide for full details on [Code Contribution Guidelines](https://github.com/patternfly/patternfly-react/blob/master/CONTRIBUTING.md#code-contribution-guidelines)
+See the PatternFly React Guide for full details on [Code Contribution Guidelines](../../CONTRIBUTING.md#code-contribution-guidelines)
```

Comment:
the link does not work... Why did you remove the contribution instructions from the guide?

Reply:
The link needs one more `../` since it's relative.